2-mandatory/1-recipes.js

```diff
@@ -22,4 +22,17 @@
   You should write and log at least 5 recipes
 */
 
-// write code here
\ No newline at end of file
+// write code here
+
+let recipe = {
+
+  title: "cake",
+  servings: 5,
+  ingredients:["milk", "eggs", "butter", "flour", "sugar"]
+  }
+
+const { title, servings, ingredients } = recipe;
```

Comment:
Using variables here is a bit overkill, as you can just write below
`console.log(recipe.title)`